pixeltable/utils/dbms_helper.py

```diff
@@ -0,0 +1,16 @@
+import abc
+
+
+class DbmsHelperBase(abc.ABC):
```

Comment:
Omit `Base` in the classname here (just `DbmsHelper` would be more conventional).

I also don't love "Helper"; how about just `Dbms` as the class name? (With implementations `CockroachDbms` and `PostgresqlDbms`, etc.)

Reply:
sounds good